### PR TITLE
Styling Cleanup / Controller Action Fix for Add Items Failures

### PR DESCRIPTION
**Pull request by JoelLindow**
1) Removed improper use of grid system and replaced with simple CSS.
2) Noticed that since we have added vendors, and added vendors to items,
you can't add items to cart. So:
  2a) Fixed multiple controller actions to pass vendor ID in params
  2b) Fixed views to receive vendor information, and confirm that you
  return to the store you were already in when adding the item.
  2c) Added vendor information to session controller action to allow
  session to know which vendor you were currently on when you added
  an item
  2d) Added an OR condition to a Vendor::Items controller so if there is
  no vendor identity in the session it will come back and be found
  through params

### CODE REVIEW

**Comment by JoelLindow** [comment]
Skipped failing tests. tests are failing due to model/schema changes and multitenancy functionality. MUST REVISIT AND MODIFY TESTS TO MEET NEW FUNCTIONALITY

**Comment by boveus** [comment]
I just pushed a few changes to this PR.  It looks like the items index had a field for @vendor.name, which was coming up as a nil since that field was not being passed down from the items index controller.  I removed that field and most of the tests seem to be passing with the exception of a few admin item tests.

I do not see a view or a test that corresponds to the vendor items path, so I am not sure where that route is supposed to go.  That route was causing some problems, though.  The code  
```ruby
 namespace :vendor, path: ':vendor', as: :vendor do
   resources :items, only: [:index]
 end
```
was causing a few problems.  Because the path is wild-carded, whenever another path had a format matching /<some_param>/items, it was redirecting it to the vendor items path.  This was causing quite a few of the admin items tests to fail, since they relied on the /admin/items path.

I move this namespace to the bottom of the routes file, so it looks for the admin/items path before looking for the <param>/items path.

I think it would be worth reviewing if we have existing tests for the features we have implemented so far, since it looks like removing the vendors name from the items index did not cause any of our existing tests to fail.  We should be writing tests for all of the features we are adding.  Please let me know if I inadvertently broken one of our features.

**Comment by JoelLindow** [comment]
^ I think these are all good points Brandon. Let's review them this afternoon, revise and we can merge this PR with the final touches.

File: app/controllers/vendor/items_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Vendor::ItemsController < ApplicationController
   def index
-    @items = Vendor.find_by(slug: params[:vendor]).items
+    @vendor = Vendor.find_by(slug: params[:vendor]) || Vendor.find(params[:vendor])
+    @items = @vendor.items
     @tags = Tag.all
```
Comment (samuelssnider) [inline app/controllers/vendor/items_controller.rb]: Nice, looks good!

**Pull request merged**